Library/src/Systems/HotspotSequence/HotspotSequenceSystem.cpp

```diff
@@ -216,6 +270,54 @@ HotspotSequenceSystem::~HotspotSequenceSystem()
 	SpaceSystem	   = nullptr;
 	SequenceSystem = nullptr;
 }
+
+void HotspotSequenceSystem::UpdateAllGroupsContainingItem(const csp::common::String& ItemName, csp::systems::NullResultCallback Callback)
```

Comment:
Can we rename this to be `RemoveItemFromGroups ` or similar to be more descriptive of what the function does?